packages/components/src/color-picker/styles.ts

```diff
@@ -81,7 +81,6 @@ export const ColorfulWrapper = styled.div`
 		align-items: center;
 		width: 216px;
 		height: auto;
-		overflow: hidden;
```

Comment:
If this style is removed, an unintended scroll bar will appear when I move the pointer to the right edge of the regular color picker. This problem can be reproduced on Windows, but it might be possible to reproduce it on MacOS by changing the scroll bar settings.

However, strangely, this problem does not occur in #55149, where [the same style has been removed](https://github.com/WordPress/gutenberg/pull/55149/files#diff-28b8d7edaea447b5f40b2b02ba5fa261ce5164360c75a067e47612276f4bd568L84).

If this PR can be merged before #55149, how about restoring this style?

https://github.com/WordPress/gutenberg/assets/54422211/e859843e-8fa6-4030-aea3-2495eed8a1df

Reply:
I think that's because it's missing the change to `packages/components/src/color-palette/index.tsx` setting `resize: false` for that specific instance of dropdown.

This can be confirmed by removing the `resize : false` change in this PR in the `packages/components/src/custom-gradient-picker/gradient-bar/control-points.tsx` file, and noting how the popover in `GradientPicker` will also start scrolling.

In short, I think this is behaving as expected.